src/elements/Header/HeaderH1.js

```diff
@@ -15,4 +15,12 @@ HeaderH1._meta = {
   type: META.type.element,
 }
 
+HeaderH1.propTypes = {
+  _sdClass: PropTypes.string,
+}
+
+HeaderH1.defaultProps = {
+  _sdClass: 'sd-header-h1',
```

Comment:
Heads up, I've finished removing all `sd-*` classes, see #301.  Rebase to master to get the update.  The test for the `sd-*` class has been inverted to fail if any `sd-` class exists.

Let's remove all `sd-*` classes from all PRs.  👍